core/src/main/kotlin/renderers/OutputWriter.kt

```diff
@@ -1,7 +1,15 @@
 package org.jetbrains.dokka.renderers
 
-interface OutputWriter{
+import org.jetbrains.dokka.CoreExtensions
+import org.jetbrains.dokka.plugability.DokkaContext
+import org.jetbrains.dokka.plugability.single
+
+interface OutputWriter {
+
+    val context: DokkaContext
```

Comment:
I think I could move this property to super interface called something like "Contextual" and make some interfaces which requires DokkaContext inherit from it. In my opinion it would be logical to enforce inclusion DokkaContext as a property of class. The question is whether `HtmlRenderer`, `DocTagToContentConverter` and `DefaultLocationProviderFactory` always need context as they need it in default implementations.

Reply:
I think this is reasonable. Thoughts @Kordyjan ?